desktop/ui/src/main/java/org/datacleaner/windows/ElasticSearchDatastoreDialog.java

```diff
@@ -224,7 +226,7 @@ public void onSelected(final FilenameTextField filenameTextField, final File fil
 
         if (originalDatastore == null) {
             _hostnameTextField.setText("localhost");
-            _portTextField.setText("9300");
+            _portTextField.setText("9200");
```

Comment:
Not a huge problem, but now I see "9200" and/or "9300" a handful of times in this class. That calls for two constants to make it easier to maintain and avoid making only partial changes.